map/map.pro

```diff
@@ -35,6 +36,8 @@ HEADERS += \
     user_mark_container.hpp \
     user_mark.hpp \
     user_mark_dl_cache.hpp \
+    api_mark_container.hpp \
```

Comment:
Feel free to sort all *.hpp and *.cpp files lexicographically in SOURCES and HEADERS blocks.